app/renderer/ui/pages/main/sections/smartContracts/tabs/marketplace/index.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from "react"
+
+import { TabInfo, TabComponent } from "app/renderer/ui/pages/main/sections"
+import { CardMarketplaceProduct } from "app/renderer/ui/components/card"
+import { marketplaceProducts } from "app/renderer/ui/pages/main/sections/wallet/MockData"
+
+const styles = require("./style.scss")
+
+export interface Iprops {
+  cover: string
+  title: string
+  author: string
+  rating: string
+  alt: string
+  tags: Array<string>
+}
+
+/**
+ * MarketPlace component
+ *
+ * @class MarketPlace
+ * @extends {TabComponent<any>}
+ */
+class MarketPlace extends TabComponent<any> {
+
+  // tslint:disable-next-line:prefer-function-over-method completed-docs
+    public render() {
+      const createMarketPlaceProduct = () => marketplaceProducts
+        .map((product) => <CardMarketplaceProduct key={product.title} {...product} />)
```

Comment:
`createMarketPlaceProduct` can be taken out of the `render()` function and likely out of `MarketPlace` too.